test_gpu.py

```diff
@@ -0,0 +1,109 @@
+import ray
+import torch
+import socket
+import os
+import torch.distributed as dist
+import time
+from ray.experimental.channel.torch_tensor_type import TorchTensorType
+from ray.experimental.channel import ChannelContext
+
+
+
+# TODOs:
+# [x] Decorator for GPU tensor returns
+# [x] Custom serializer / deserializer to get value from actor store
+# [ ] Return torch tensor metadata to driver when task finishes
+# [x] register collectives in python
+# [ ] On task submission, driver sends send/recv tasks to actors A and B
+
+
+WORLD_SIZE = 2
+
+@ray.remote
+class Actor:
+
+    def register_custom_serializer(self):
```

Comment:
We should have a one-line call from the driver that enables the custom serialization.

Reply:
Maybe can do this as part of the `ray.init_communicator` call.